apps/odyssey-stats/src/widget/highlights.jsx

```diff
@@ -0,0 +1,71 @@
+import classNames from 'classnames';
+import { useTranslate } from 'i18n-calypso';
+import moment from 'moment';
+import useReferrersQuery from '../hooks/use-referrers-query';
+import useTopPostsQuery from '../hooks/use-top-posts-query';
+
+import './hightlights.scss';
+
+function TopColumn( { items, viewAllUrl, viewAllText, title, className = null } ) {
```

Comment:
For a follow-up PR: This component would benefit from loading state handling. Currently, it says "Sorry, nothing to report"; we can have it say something like "Loading..." instead.